### PR TITLE
feat(grid): remove unsupported sorting options on snippets page

### PR DESCRIPTION
**Pull request by SunsetTechuila**

### CODE REVIEW

File: src/components/Grid.tsx [hunk 1]
```diff
@@ -546,11 +546,16 @@ class Grid extends React.Component<
...
-              sortBySelectedFn={(a) => a.key === this.CONFIG.sort} />
+            {this.CONFIG.activeTab !== "Snippets" ? (
+              <>
+                <h2 className="marketplace-header__label">{t("grid.sort.label")}</h2>
+                <SortBox
+                  onChange={(value) => this.updateSort(value)}
+                  sortBoxOptions={generateSortOptions(t)}
+                  sortBySelectedFn={(a) => a.key === this.CONFIG.sort} />
+              </>
+            ) : <h1>{this.props.title}</h1>
```
Comment (theRealPadster) [inline src/components/Grid.tsx:557]: Is the `this.props.title` the "Spicetify Marketplace text? I removed that in the original PR.

**Comment by theRealPadster** [comment]
The sort does work on the snippets page, it just only supports sorting by name. I'm not sure what the best solution is here, since I can see it being useful as there are a lot of snippets now.

File: src/components/Grid.tsx (hunk 1 above)
Comment (SunsetTechuila) [reply src/components/Grid.tsx:557]: > I removed that in the original PR

I know, but without it header looks empty

Comment (theRealPadster) [reply src/components/Grid.tsx:557]: Still, doesn't make sense to have a title only show up on a single tab.

**Comment by SunsetTechuila** [comment]
> it just only supports sorting by name

does it? then please wait

**Comment by SunsetTechuila** [comment]
oh, now it resets

**Comment by SunsetTechuila** [comment]
ok, then I don't know what to do without creating kludges

**Pull request closed without merging**